src/app/store/actions/ReferenceData.actions.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { Preparer } from '@app/models/preparer';
+import { TestStation } from '@app/models/test-station';
+import {
+  EReferenceDataActions,
+  LoadPreparers,
+  LoadPreparersSuccess,
+  LoadTestStations,
+  LoadTestStationsSuccess
+} from '@app/store/actions/ReferenceData.actions';
+
+describe('LoadPreparers', () => {
+  test('action should have correct type', () => {
```

Comment:
it is better if we stick with it for consistency.
Also, we don't need action maybe we can have some description, like
it('should have the correct action type when LoadPreparers() action is instantiated').